spec/integration/describe_spec.rb

```diff
@@ -0,0 +1,6 @@
+RSpec.describe Regexp do
+
+  it 'sets the core class as the described class' do
+    expect(described_class).to eq(Regexp)
+  end
+end
```

Comment:
Do you mind putting a spec here instead?

https://github.com/rspec/rspec-core/blob/229bbaa1093aeba3e7c9c02eb9fc82fb36d5c2a3/spec/rspec/core/metadata_spec.rb#L265-L281

That'll keep the specs for `described_class` together.